src/components/ScaleBar.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useContext } from "react";
+
+import styles from "./scalebar.module.css";
+import { MICRO } from "../constants";
+import { SimulariumContext } from "../simulation/context";
+import { AGENT_AB_COLOR } from "../constants/colors";
+
+const ScaleBar: React.FC = () => {
+    const { maxConcentration } = useContext(SimulariumContext);
+    const labelArray = [];
+    for (let i = maxConcentration; i >= 0; i = i - 2) {
+        labelArray.push(i);
+    }
+    return (
+        <div className={styles.container}>
```

Comment:
Consider adding an `img` role to this div and describing the scale bar it contains with `aria-label`?


e.g.
```suggestion
        <div className={styles.container} role="img" aria-label={`Color key for the cuvette. The lowest concentration (white) is 0 ${MICRO}M and the highest concentration (yellow) is ${maxConcentration} ${MICRO}M.`}>
```

See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role